File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/mix/MixPositionApi.java
package com.bitget.openapi.api.mix;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Contract position api
 */
public interface MixPositionApi {

    /**
     * Obtain single contract position information
     * @param symbol
     * @param marginCoin
     * @return ResponseResult
     */
    @GET("/api/mix/v1/position/singlePosition")
    Call<ResponseResult> singlePosition(@Query("symbol") String symbol,
                                        @Query("marginCoin") String marginCoin);

    /**
     * Obtain all contract position information
     * @param productType
     * @param marginCoin
     * @return ResponseResult
     */
    @GET("/api/mix/v1/position/allPosition")
    Call<ResponseResult> allPosition(@Query("productType") String productType,
                                     @Query("marginCoin") String marginCoin);
}

File: bitget-php-sdk-api/src/api/mix/MixPositionApi.php
<?php


namespace bitget\api\mix;


use bitget\internal\BitgetApiClient;

class MixPositionApi
{
    const BASE_URL = "/api/mix/v1/position";

    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }
    /**
     * Obtain single contract position information
     * @param $symbol
     * @param $marginCoin
     * @return string
     */
    public function singlePosition(string $symbol,string $marginCoin):string
    {
        $params = array("symbol" => $symbol, "marginCoin" => $marginCoin);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/singlePosition", $params);
    }
    /**
     * Obtain all contract position information
     * @param $productType
     * @param $marginCoin
     * @return string
     */
    public function allPosition(string $productType,string $marginCoin):string
    {
        $params = array("productType" => $productType, "marginCoin" => $marginCoin);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/allPosition", $params);
    }

}